marsha/core/tests/test_api_upload_confirm.py

```diff
@@ -34,15 +37,43 @@ def test_api_upload_confirm_video(self):
         self.assertEqual(video.uploaded_on, datetime(2018, 8, 8, tzinfo=pytz.utc))
         self.assertEqual(video.state, "ready")
 
+    @override_settings(
+        UPLOAD_CONFIRM_SHARED_SECRETS='["dummy secret", "actual secret"]'
```

Comment:
call them previous_secret, current_secret to convey the reason of having many secrets? Dummy sends a wrong info.